chemicaltoolbox/autodock_vina/docking/docking.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="docking" name="Docking" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.1.2">autodockvina</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <description> - Tool for running docking with Autodock Vina</description>
+    <command interpreter="python"><![CDATA[
+        docking.py -l "$ligand" 
+                   -r "$receptor" 
+                   -b "$box" 
+                   -p "$file_output1.files_path" 
+                   -f "$file_output1" 
+                   -g "$file_output2"
+    ]]></command>
+    <inputs>
+        <param type="data" name="ligand" format="pdbqt" label="Ligands: " help="No data? See tip below" />
+        <param type="data" name="receptor" format="pdbqt" label="Receptor: " help="No data? See tip below" />
+        <param type="data" name="box" format="txt" label="Binding site coord.: " help="No data? See tip below" />
+    </inputs>
+    <outputs>
+        <data name="file_output1" format="pdbqt" label="#echo (os.path.splitext (str($ligand.name))[0]) + '_-_' + (os.path.splitext (str($receptor.name))[0])#.pdbqt" />
+        <data name="file_output2" format="log" label="#echo (os.path.splitext (str($ligand.name))[0]) + '_-_' + (os.path.splitext (str($receptor.name))[0])#.log" />
```

Comment:
Is this really a molecule format. From the test data I think this is just a normal text file. You can use `txt` for it and remove the additional file type for LOG.
I would actually not output it but pipe it to stdout. The advanced Galaxy user can see it there and we do not pollute the history with for every other user. What do you think?